gh-95601: restore support for awaitable objects that are not futures in `asyncio.wait` (#95708)
python/cpython: Misc/NEWS.d/next/Library/2022-08-05-11-16-33.gh-issue-95601.hJvI9y.rst

> The explicit passing of awaitable objects that are not futures to :func:`asyncio.wait` has been deprecated and will be removed in version 3.14.

Reviewer: ```suggestion
The explicit passing of awaitable objects that are not :class:`asyncio.Future` to :func:`asyncio.wait` is deprecated and will be removed in version 3.14.
```

> The explicit passing of awaitable objects that are not :class:`asyncio.Future` to :func:`asyncio.wait` is deprecated and will be removed in version 3.14.

Author: ```suggestion
Restore the support for the explicit passing of awaitable objects that are not :class:`asyncio.Future` to :func:`asyncio.wait`
```
